src/win/util.c

```diff
@@ -1380,6 +1380,7 @@ int uv__getpwuid_r(uv_passwd_t* pwd) {
   }
 
   pwd->shell = NULL;
+  pwd->gecos = NULL;
```

Comment:
For ease of use, it's perhaps better to make it an empty string instead of a NULL pointer.  Just a thought.